avocado/core/job.py

```diff
@@ -371,31 +370,15 @@ def _log_avocado_datadir():
         job_log.info('logs     ' + data_dir.get_logs_dir())
         job_log.info('')
 
-    def _log_variants_tree(self, variant):
```

Comment:
Sure, I was just asking about the balance between nice and long-to-deliver and we already solved this on meeting. No doubts about this anymore 😄